### PR TITLE
Add from and size documentation in listCollections

### PR DESCRIPTION
**Pull request by dbengsch**

### CODE REVIEW

**Comment by dbengsch** [comment]
fix #57

File: source/includes/_readController.md [hunk 1]
```diff
@@ -193,27 +199,24 @@ Only documents in the persistent data storage layer can be retrieved.
...
-        'collection_...',
-        'collection_n'
-      ]
-    },
+    collections: [
+      {name: 'realtime_1', type: 'realtime'},
+      {name: 'realtime_2', type: 'realtime'},
+      {name: 'realtime_...', type: 'realtime'},
+      {name: 'realtime_n', type: 'realtime'},
+      {name: 'sorted_1', type: 'sorted'},
```
Comment (stafyniaksacha) [inline source/includes/_readController.md]: nitpicking: `sorted` or `stored` ?

Comment (dbengsch) [inline source/includes/_readController.md]: @stafyniaksacha right, it is solved.

File: source/includes/_readController.md [hunk 2]
```diff
@@ -193,27 +199,24 @@ Only documents in the persistent data storage layer can be retrieved.
...
+      {name: 'stored_...', type: 'stored'},
+      {name: 'stored_n', type: 'stored'}
+    ],
     type: 'all'
   }
 }
 ```
 
-Return the complete list of realtime and stored  data collections in requested index.  
+Return the complete list of realtime and stored data collections in requested index sorted by name in a alphanumerical order.
```
Comment (benoitvidis) [inline source/includes/_readController.md]: sorted by name in ~~a~~ _alphanumeric_ order.

Comment (dbengsch) [inline source/includes/_readController.md]: @benoitvidis done

**Pull request merged**